app/views/bodies/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%= @body.name %></h1>
+
+<p><%= "#{@body.name} was discovered by #{@body.discovered_by} on #{discovery_date}." %></p>
```

Comment:
You will need to write an if statement here, as not all bodies have discovered_by and discoverey_date. You will also need body.discovery_date and specify how you would like that to be formatted, as it is a date object.